### DISCUSSION TITLE
Add New Website Link in the About Section

### DISCUSSION THREAD

**Discussion by surjithctly** [alpinejs/alpine#1644] [3. Feature ideas]
It would be easy to navigate if you put the new site https://alpinejs.dev/ in the About Section as well.

![image](https://user-images.githubusercontent.com/1884712/123508211-d1adea00-d622-11eb-9a8d-b14ccd9cd004.png)

**Comment by calebporzio** [comment]
![image](https://user-images.githubusercontent.com/3670578/123522147-867af200-d689-11eb-844e-fb75f9234e6d.png)
Done, thanks!
